README.md

```diff
@@ -60,6 +60,41 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 Done!
 
+### Semi Dockerized Setup
```

Comment:
This also shares a bunch of steps from the above (steps 1-7) so maybe we should break this into something like

- General setup (fully dockerized steps 1-7)
  - Fully dockerized (fully dockerized steps 8-9)
  - Demi dockerized (these steps)